index.html

```diff
@@ -0,0 +1,98 @@
+				</section>
+				</div>
+				
+				    <section class="portfolio">
+						
+					<div class="portfolio-piece1"> 
+						<main class="container">
+							<h2 class="portfolio-text-head">Challenge 1 - Make a logo just using HTML and CSS</h2>
+								<p class="portfolio-text">The challenge was to make a logo just using CSS and HTML</p>
+									<img class="portfolio-image" src="logo.png" alt="A picture of a badly designed logo">
+						</main>		
+					</div>
+							
+					<div class="portfolio-piece2">
+						<main class="container">
+							<h2 class="portfolio-text-head">Project 1</h2>
+								<p class="portfolio-text">Who knows what will happen</p>
+									<img class="portfolio-image" src="construction-sign-label.avif" alt="under construction sign">
+						</main>	
+					</div>
+						
+					<div class="portfolio-piece3">
+						<main class="container">
+							<h2 class="portfolio-text-head">Project 2</h2>
+								<p class="portfolio-text">Who knows what will happen</p>
+									<img class="portfolio-image" src="construction-sign-label.avif" alt="under construction sign">
+						</main>			
+					</div>
+						
+					<div class="portfolio-piece4">
+						<main class="container">
+							<h2 class="portfolio-text-head">Project 3</h2>
+								<p class="portfolio-text">Who knows what will happen</p>
+									<img class="portfolio-image" src="construction-sign-label.avif" alt="under construction sign">
+						</main>	
+					</div>
+					
+					<div class="portfolio-piece5">
+						<main class="container">
+							<h2 class="portfolio-text-head">Project 4</h2>
+								<p class="portfolio-text">Who knows what will happen</p>
+									<img class="portfolio-image" src="construction-sign-label.avif" alt="under construction sign">
+						</main>	
+					</div>
+				</section>
+					
+					
+			<div id="contact" class="contact-catcher"></div>
+				<section class="contact">
+						<h2 class="contact">Contact information</h2>
+					<section class=contact-tidy>
+							<p class="contact-intro">If you would ever like to be involved in the calming chaos:</p>
+							<p class="contact">Email: thejoshsliasbryan@gmail.com</p>
+							<p class="contact">Telephone: 07584093030</p>
+							<p class ="contact">LinkedIn<a class="contact" target="_blank" 								href="https://uk.linkedin.com/in/joshua-bryan-2aab3a37?trk=people-guest_people_search-card&challengeId=AQFS4fXC41MILwAAAYlJK1e1yY0YgbPGEmNWy6EBCKh5vGf38mh_PUaU										rVBUaEzCiYcIUrhrGUxO1jjTRNp1_xpg9yD_avrq-A&submissionId=0ad11385-b910-7117-7fa4-87a3df2d6e6c&challengeSource=AgGqcOW-5ofE1wAAAYlJK6BvhG4ZmTQQZBpOyYs											CW2rjbckKrLgBhfqNw5bej-U&challegeType=AgHNJXySRe9wjgAAAYlJK6B0vVCSIde_vLErVCjJ0CT8LecOheJz9Fo&memberId=AgFJy0WxlQR-NgAAAYlJK6B4Ni6JTlyPJs6wZCuqd											dGaC-0&recognizeDevice=AgE42_VVzFJ-WAAAAYlJK6B8hxptpZIm3fEipA_es7W3OG5dJnBl">: Josh Bryan</a></p>
```

Comment:
here again we have a lot of different elements that all have the class "contact", it would be better if that class was only used on one of those elements and the rest either be selected using other means, or given their own more specific classes